pypeit/spectrographs/keck_deimos.py

```diff
@@ -741,8 +763,8 @@ def get_rawimage(self, raw_file, det):
         mosaic = None if nimg == 1 else self.get_mosaic_par(det, hdu=hdu)
         detectors = [self.get_detector_par(det, hdu=hdu)] if nimg == 1 else mosaic.detectors
 
-        if hdu[0].header['AMPMODE'] != 'SINGLE:B':
-            msgs.error('PypeIt can only reduce images with AMPMODE == SINGLE:B.')
+        if hdu[0].header['AMPMODE'] not in ['SINGLE:B', 'SINGLE:A']:
+            msgs.error('PypeIt can only reduce images with AMPMODE == SINGLE:B or AMPMODE == SINGLE:A.')
```

Comment:
Ah, I see that the `DUAL:A+B` mode is handled here. If the two places mentioned above are downstream of this, please ignore them.